<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(41, 92, 92, 200)</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QPushButton" name="save_button">
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgba(0,0, 0, 0);
border: 2px solid rgba(127, 57, 91, 255);
border-style: inset;
border-radius: 5px;
padding: 12px;
font-size: 15pt;
color: rgba(255, 255, 255, 230);
margin-left: 10px;
margin-right: 10px;
}

QPushButton:hover {
background-color: qconicalgradient(cx:0.431909, cy:1, angle:270.1, stop:0 rgba(0, 0, 0, 255), stop:0.39548 rgba(191, 112, 151, 100));
}</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="add_one_more">
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgba(0,0, 0, 0);
border: 2px solid rgba(127, 57, 91, 255);
border-style: inset;
border-radius: 5px;
padding: 12px;
font-size: 15pt;
color: rgba(255, 255, 255, 230);
margin-left: 10px;
margin-right: 10px;
}

QPushButton:hover {
background-color: qconicalgradient(cx:0.431909, cy:1, angle:270.1, stop:0 rgba(0, 0, 0, 255), stop:0.39548 rgba(191, 112, 151, 100));
}</string>
      </property>
      <property name="text">
       <string>Добавить еще одну заметку</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(191, 112, 151, 200);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit {
border: 2px solid rgba(255, 255, 255, 255);
font-size: 16pt;
color: white;
background-color: rgba(41, 92, 92, 200);
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgba(0,0, 0, 0);
border: 2px solid rgba(127, 57, 91, 255);
border-style: inset;
border-radius: 5px;
padding: 12px;
font-size: 15pt;
color: rgba(255, 255, 255, 230);
margin-left: 100px;
margin-right: 100px;
}

QPushButton:hover {
background-color: qconicalgradient(cx:0.431909, cy:1, angle:270.1, stop:0 rgba(0, 0, 0, 255), stop:0.39548 rgba(191, 112, 151, 100));
}</string>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QFrame" name="frame_6">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(191, 112, 151, 200);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-family: 'Arial' 'Optima';
font: bold;
font-size: 15pt;
color: rgba(255, 255, 255, 255);
margin-left: 3px;
background-color: rgba(41, 92, 92, 200);
}</string>
       </property>
       <property name="text">
        <string>Заметка</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(191, 112, 151, 200);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-family: 'Arial' 'Optima';
font: bold;
font-size: 15pt;
color: rgba(255, 255, 255, 255);
margin-left: 3px;
background-color: rgba(41, 92, 92, 200);
}</string>
       </property>
       <property name="text">
        <string>Название путешествия</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="1" rowspan="2">
     <widget class="QFrame" name="frame_7">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(191, 112, 151, 200);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>331</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit {
border: 2px solid rgba(255, 255, 255, 255);
font-size: 16pt;
color: white;
background-color: rgba(41, 92, 92, 200);
}</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(191, 112, 151, 200);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
font-family: 'Arial' 'Optima';
font: bold;
font-size: 15pt;
color: rgba(255, 255, 255, 255);
margin-left: 3px;
background-color: rgba(41, 92, 92, 200);
}</string>
       </property>
       <property name="text">
        <string>Город, место на карте</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_5">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(191, 112, 151, 200);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="name_travel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
border: 2px solid rgba(255, 255, 255, 255);
font-size: 16pt;
color: white;
background-color: rgba(41, 92, 92, 200);
}</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
